packages/core/src/App/Components/Layout/Header/dtrader-v2/real-signup-banner-dtrader-v2.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import classNames from 'classnames';
+import 'Sass/app/_common/components/banner-dtrader-v2.scss';
+import { Text } from '@deriv-com/quill-ui';
+import {
+    CurrencyBtcIcon,
+    CurrencyEthIcon,
+    CurrencyEurIcon,
+    CurrencyUsdIcon,
+    LabelPairedChevronRightMdRegularIcon,
+} from '@deriv/quill-icons';
+import { Localize } from '@deriv/translations';
+
+type TRealSignupBannerDTraderV2 = {
+    openRealAccount: () => void;
+    is_eu?: boolean;
+};
+const RealSignupBannerDTraderV2 = ({ openRealAccount, is_eu }: TRealSignupBannerDTraderV2) => (
+    <button className='banner__container' onClick={openRealAccount}>
+        <div className='banner'>
+            <div
+                className={classNames('banner__currency', {
+                    'banner__currency--is-eu': is_eu,
+                })}
+            >
+                {is_eu ? (
+                    <React.Fragment>
+                        <CurrencyEurIcon iconSize='md' className='icon-1' />
+                        <CurrencyUsdIcon iconSize='md' className='icon-2' />
+                    </React.Fragment>
+                ) : (
+                    <React.Fragment>
+                        <CurrencyBtcIcon iconSize='md' className='icon-1' />
+                        <CurrencyEthIcon iconSize='md' className='icon-2' />
+                        <CurrencyUsdIcon iconSize='md' className='icon-3' />
+                    </React.Fragment>
+                )}
```

Comment:
```suggestion
        {is_eu
            ? banner_icons.eu.map((icon, idx) => {
                  return <icon key={idx} iconSize='md' className='banner__icon' />;
              })
            : banner_icons.row.map((icon, idx) => {
                  return <icon key={idx} iconSize='md' className='banner__icon' />;
              })}
```
For this to work, you will need the below config: :)

```
const banner_icons = {
    eu: [CurrencyEurIcon, CurrencyUsdIcon],
    row: [CurrencyBtcIcon, CurrencyEthIcon, CurrencyUsdIcon],
};
```
*styles for specific icons can be handled using `:nth-child()`

What do you think about this suggestion? :)

Reply:
Niiiice, thank you, will try